tests/suites/test_suite_psa_crypto.data

```diff
@@ -6118,7 +6118,7 @@ depends_on:PSA_WANT_ALG_HKDF:PSA_WANT_ALG_SHA_256:PSA_WANT_ALG_TLS12_PRF
 derive_key_exercise:PSA_ALG_TLS12_PRF(PSA_ALG_SHA_256):"0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b":"000102030405060708090a0b0c":"f0f1f2f3f4f5f6f7f8f9":PSA_KEY_TYPE_DERIVE:400:PSA_KEY_USAGE_DERIVE:PSA_ALG_HKDF(PSA_ALG_SHA_256)
 
 PSA key derivation: HKDF-SHA-256 -> ECC secp256r1, exercise ECDSA
-depends_on:PSA_WANT_ALG_ECDSA:PSA_WANT_ALG_HKDF:PSA_WANT_ALG_SHA_256:PSA_WANT_KEY_TYPE_ECC_KEY_PAIR:PSA_WANT_ECC_SECP_R1_256
+depends_on:PSA_WANT_ALG_ECDSA:PSA_WANT_ALG_HKDF:PSA_WANT_ALG_SHA_256:PSA_WANT_KEY_TYPE_ECC_KEY_PAIR:PSA_WANT_ECC_SECP_R1_256:MBEDTLS_ECP_C
```

Comment:
Note: if indeed we need the full `ECP_C`, then we'll also want to expand the ignore list in `analyze_outcomes.py` as it's currently failing.

Reply:
I think that when I looked at this I was erroneously confused by the `mbedtls_mpi_` calls that reminded me of arithmetic, so I kept the `ECP_C` dependency. In facts this is not the case since they come from `MPI` and not `ECP`.

Reply:
Yes, the naming "arithmetic" is an approximation, what we really mean here everything associated with scalar multiplication - that is, `ecp_mul()` and everything that either uses it or is only used to implement it.

There are things we keep with `ECP_LIGHT && !ECP_C` that still do computations, for example:
- private key validation does bignum comparisons (less than, greater than);
- public key validation computes Y^2 mod P and X^3 + a X + b mod P and compares for equality;
- reading compressed points computes X^3 + aX + b mod P and takes the square root mod P.

So, saying we remove "arithmetic" is really sloppy wording but I couldn't find something that was short enough for daily use while still being accurate ;)

Anyway, when we have `ECP_LIGHT` we have `BIGNUM_C`. By the way, that's probably something we should be enforcing in `check_config.h`.

Reply:
> then we'll also want to expand the ignore list in analyze_outcomes.py as it's currently failing

I suspect I'll have to. I replaced `ECP_C` with `ECP_LIGHT` in key derivation, but still `analyze_outcomes` complains for disparities. Actually `ecp` is not the only one, so I'll add it to the ignore list and fix the others.

Reply:
Done, now also `analyze_outcomes.py` should be fine ;)